core/storage/classifier/gae_models.py

```diff
@@ -262,8 +293,34 @@ def create(
         if not cls.get_by_id(instance_id):
             mapping_instance = cls(
                 id=instance_id, exp_id=exp_id, exp_version=exp_version,
-                state_name=state_name, classifier_id=classifier_id)
+                state_name=state_name, job_id=job_id)
 
             mapping_instance.put()
             return instance_id
         raise Exception('A model with the same ID already exists.')
+
+    @classmethod
+    def create_multi(cls, mapping_dicts_list):
+        """Creates multiple new  TrainingJobExplorationMappingModel entries.
+
+        Args:
+            mapping_dicts_list: list(dict). The list of dicts where each dict
+                represents the attributes of one
+                TrainingJobExplorationMappingModel.
+        """
+        list_mapping_models = []
```

Comment:
The arg has _list at the end; these have list_ at the beginning -- seems like an inconsistency.

I recommend just calling these `mapping_models` and `mapping_ids`.